### ISSUE TITLE
How to make myself facemodel_info data?

### ISSUE THREAD

**Issue by nxsf8023** [ascust/3DMM-Fitting-Pytorch#10]
Now I have a project on 3DMM re. I want to us BFM2019FaceModel and make my own facemodel_info.
May I get some way? THX

**Comment by ascust** [comment]
It is possible to get your own facemodel_info file. The only difficulty is the correspondence between the 3D vertices and 2D landmarks. The remaining part of the info file is only about shape basis, expression basis or texture basis, which are basically some matrices.

**Comment by nxsf8023** [comment]
Now I am trying to use BFM2019 to make my own face info file.
Could I use Farkas_face05.fp to make a 68landmarks data?

**Comment by ascust** [comment]
I think the current version of the code would not give you the correct file. You could try to follow "convert_bfm09_data.py" and generate the corresponding parameters of the dict like "idBase", "exBase", "texBase", etc. I will consider adding support for BFM2019 in the future.

**Comment by nxsf8023** [comment]
I have do some work as you say. I make myself convert file to gen my own BFM2019/BFM2017 model but if I want train the model I must use new keypoints data. Otherwise the loss can't be calculated. So I do my work on this. Thx  for your quick reply!

**Comment by ascust** [comment]
well, give it a try. Feel free to discuss if you have any questions.
